charmcraft.yaml

```diff
@@ -6,8 +6,6 @@ bases:
     run-on:
     - name: "ubuntu"
       channel: "22.04"
-    - name: "ubuntu"
```

Comment:
Is this intended?

Reply:
This is a Kubernetes charm, I think the series is not very important. So I just remove the `20.04` to prevent this from happening again.